README.md

```diff
@@ -164,17 +164,46 @@ We should reasonably crop images if needed and then scale them to be EITHER:
 - 1000px wide if it's a landscape image
 - 600px wide if it's a portrait image
 
-Make sure to export it as a `.jpg` image at a **quality level of 70**, which should ensure a reasonable
+Make sure to export it as a `.webp` image at a **quality level of 70**, which should ensure a reasonable
 file size under 200 kB.
 
-\*\*Store the image in `/static/building-imgs/`.
+**Store the image in `/static/building-imgs/`.**
 
 3. **Tell The Site There's a Building Image** - Follow the pattern of other buildings in the
    `building-images.constant.vue`, providing an attribution URL, the image file name, and specify
    whether it's a tall (portrait) image and whether it's from Google Maps.
 
 4. **Confirm the image is visible and looks good** - and that's all there is to it!
 
+### Automatically Fetching Images from Streetview
+
+We have a handy script, `fetch_streetview_imagery.py` that can be used to automatically fetch
+Streetview building images by address. Unfortunately, Streetview imagery can be pretty hit or miss,
+and so this is still a pretty manual process.
+
+
+1. **Get a Google Maps API Key** - Ask Viktor for a key or setup your only Google Maps API key with
+Google
+
+2. **Create An Addresses CSV** - create a CSV file of buildings addresses you want to fetch, with
+columns:
+
+- **address** - the full address (e.g "10 W 31st Street, Chicago IL, 60616")
```

Comment:
I'm considering also requiring the building ID, so that can be in the filename, saving searches.

I could then also automatically generate some output code for the custom images constant 🤔